riscv/runtime/src/lib.rs

```diff
@@ -0,0 +1,21 @@
+
+#![no_std]
+
+use core::arch::asm;
+use core::panic::PanicInfo;
+
+#[panic_handler]
+fn panic(_panic: &PanicInfo<'_>) -> ! {
+    // activate after #176 has been printed
+    // print(format!("panic at line {} in file {}\n", panic.location().line(), panic.location().file()));
+    loop {}
```

Comment:
```suggestion
    unsafe {
        asm!("unimp");
    }
    loop {}
```
without the `unimp`, it will succeed in generating a proof, but I guess we want proofs to be invalid if they panic.